README.md

```diff
@@ -0,0 +1,6 @@
+# Airshipper
```

Comment:
There's generally no need to provide README as users are unlikely to ever see it.

Reply:
Removed the README.